app/uk/gov/hmrc/plasticpackagingtax/registration/controllers/group/OrganisationDetailsTypeController.scala

```diff
@@ -62,7 +62,8 @@ class OrganisationDetailsTypeController @Inject() (
 
   def displayPage(): Action[AnyContent] =
     (authenticate andThen journeyAction).async { implicit request =>
-      Future(Ok(page(form = OrganisationType.form())))
+      val isFirstMember: Boolean = request.registration.groupDetail.exists(_.isFirstMember)
```

Comment:
Might read better to add a method to registration, like:
```
def groupMembersPresent() = groupDetail.exists(!_.members.isEmpty)
```

Reply:
Done